src/components/auth/signup/signup.html

```diff
@@ -20,12 +20,12 @@ <h1 class="no-margin">S3 Portal</h1>
           type="email" name="email" required email placeholder="Email"
           ng-model="signup.credentials.email"
           ng-readonly="signup.form.$submitted"
-          ng-blur="signup.checkEmail()"
+          ng-change="signup.checkEmail()"
```

Comment:
The change event is fired when user changes the value, so many check email requests will be sent.
It is better to use delayed request here. For example, user dose not change the value in 1000ms, so send the request.